<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="w">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>513</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>w, рад</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="fi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>553</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>fi, рад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>570</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="F">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>553</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>F, Н</string>
   </property>
  </widget>
  <widget class="QWidget" name="drawingArea" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid black</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>501</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>514</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="N">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>531</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>2.000000000000000</double>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="k">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>530</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>513</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>k, Н/м</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="t">
   <property name="geometry">
    <rect>
     <x>679</x>
     <y>533</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>679</x>
     <y>516</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>N</tabstop>
  <tabstop>F</tabstop>
  <tabstop>w</tabstop>
  <tabstop>fi</tabstop>
  <tabstop>k</tabstop>
  <tabstop>t</tabstop>
  <tabstop>btn_start</tabstop>
  <tabstop>btn_pause</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
